Ghidra/Framework/Graph/src/main/java/ghidra/graph/viewer/layout/VisualGraphLayout.java

```diff
@@ -64,10 +63,9 @@
 	 * need not loop over all edges to determine if they have articulations.  (Looping over
 	 * large graphs is time intensive.)
 	 * 
-	 * <P>{@link VisualEdge#articulations What are articulations?}</A>
+	 * <P><A HREF="../VisualEdge.html#articulations">What are articulations?</A>
```

Comment:
Directly using an html link is required since articulations isn't a member of VisualEdge.

Reply:
This does not work with the Eclipse hover javadoc feature.  I'll have to ask @dragonmacher about the best way to do it.